### PR TITLE
Login screen

### PR DESCRIPTION
**Pull request by HoangChu-Claremont**
User can login / sign up / logout 

TODO: Using FacebookSDK to login by Facebook 
Current Error in Build: "FAILURE: Build completed with 8 failures. 
1: Task failed with an exception. ----------- * What went wrong: Execution failed for task ':app:checkDebugAarMetadata'. 
> Could not resolve all files for configuration ':app:debugRuntimeClasspath'. 
> Could not find any version that matches com.github.parse-community.Parse-SDK-Android:facebook:latest.version.here. Versions that do not match: 
..."

https://user-images.githubusercontent.com/65592963/174920974-4d85bd5c-10d4-4d68-b2dc-67af42487eb6.mp4

https://user-images.githubusercontent.com/65592963/174920978-70591449-4e25-4746-abae-93bfdba8cdd5.mp4

https://user-images.githubusercontent.com/65592963/174920981-a6e633e0-1361-47a6-aa96-841b54af85d8.mp4

https://user-images.githubusercontent.com/65592963/174920983-4309dc83-d00d-4457-90a2-52f651de3917.mp4

https://user-images.githubusercontent.com/65592963/174920986-fc4b7cfb-5d0d-4168-9c04-a8f5729845c6.mp4

https://user-images.githubusercontent.com/65592963/174920989-64c9595f-3af8-48da-b9ea-c02aa5c1e80c.mp4

https://user-images.githubusercontent.com/65592963/174920993-6bd7b9e3-e2b8-4ede-9e0c-b857005848c4.mp4

### CODE REVIEW

File: app/src/main/java/com/example/where_to_go/MainActivity.java [hunk 1]
```diff
@@ -2,13 +2,40 @@
...
 import android.os.Bundle;
+import android.util.Log;
 
-public class MainActivity extends AppCompatActivity {
+import com.parse.ParseUser;
+
+public class MainActivity extends AppCompatActivity{
+
+    private static final String TAG = "MainActivity";
+    Intent i; // Navigate to either login or home screen
```
Comment (jinsoo601) [inline app/src/main/java/com/example/where_to_go/MainActivity.java:14]: any reason this has to be a class property here? Can you just declare intent variable in `goHomeActivity` or `goLoginActivity`? Class property is generally used only for
1. constants
2. A piece of data to represent current state of the class

Intent in this case is just used within `goXXXActivity` functions, so it is neither of those cases.

File: app/src/main/java/com/example/where_to_go/ParsApp.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import com.parse.ParseObject;
+
+public class ParsApp extends Application {
+    @Override
+    public void onCreate() {
+        super.onCreate();
+
+        Parse.initialize(new Parse.Configuration.Builder(this)
+                .applicationId(getString(R.string.back4app_app_id))
+                // if defined
```
Comment (jinsoo601) [inline app/src/main/java/com/example/where_to_go/ParsApp.java:14]: commenting is good, but this is not needed :) remove this

**Review by jinsoo601 — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/where_to_go/LoginActivity.java [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+import android.widget.EditText;
+import android.widget.Toast;
+
+//import com.facebook.CallbackManager;
+//import com.facebook.FacebookCallback;
+//import com.facebook.FacebookException;
+//import com.facebook.GraphRequest;
+//import com.facebook.GraphResponse;
+//import com.facebook.login.LoginResult;
+//import com.facebook.login.widget.LoginButton;
```
Comment (anorth2) [inline app/src/main/java/com/example/where_to_go/LoginActivity.java:19]: nit: generally don't leave commented out code as it makes files larger and harder to read.

File: app/src/main/res/layout/activity_login.xml [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="168dp"
+        android:layout_weight="1"
+        android:ems="10"
+        android:hint="@string/username"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.0"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
```
Comment (anorth2) [inline app/src/main/res/layout/activity_login.xml:22]: nit: generally order these so properties with the same prefix like `layout_` are together.

File: app/src/main/java/com/example/where_to_go/MainActivity.java (hunk 1 above)
Comment (HoangChu-Claremont) [reply app/src/main/java/com/example/where_to_go/MainActivity.java:14]: I thought having a class property avoids duplications. Lesson learned!

**Pull request merged**